Pod/Classes/ios/NYTPhotosViewController.m

```diff
@@ -239,8 +240,32 @@ - (void)actionButtonTapped:(id)sender {
                 [self.delegate photosViewController:self actionCompletedWithActivityType:activityType];
             }
         };
-        
-        [self presentViewController:activityViewController animated:YES completion:nil];
+
+        [self displayActivityViewController:activityViewController animated:YES];
+    }
+}
+
+- (void)displayActivityViewController:(UIActivityViewController *)controller animated:(BOOL)animated {
+
+    if(UI_USER_INTERFACE_IDIOM() == UIUserInterfaceIdiomPhone) {
```

Comment:
Can you update the `if` statements in this method to have whitespace between the `if` and the `(` to conform to the [Times Objective-C Style Guide](https://github.com/NYTimes/objective-c-style-guide)?

Thanks!